simulator/simulator_compatibility_test/package.xml

```diff
@@ -9,14 +9,14 @@
   <maintainer email="shumpei.wakabayashi@tier4.jp">Shumpei Wakabayashi</maintainer>
   <license>TODO: License declaration</license>
 
-  <depend>autoware_auto_control_msgs</depend>
-  <depend>autoware_auto_geometry_msgs</depend>
-  <depend>autoware_auto_mapping_msgs</depend>
-  <depend>autoware_auto_msgs</depend>
-  <depend>autoware_auto_perception_msgs</depend>
-  <depend>autoware_auto_planning_msgs</depend>
-  <depend>autoware_auto_system_msgs</depend>
-  <depend>autoware_auto_vehicle_msgs</depend>
+  <depend>autoware_control_msgs</depend>
+  <depend>autoware_geometry_msgs</depend>
+  <depend>autoware_map_msgs</depend>
+  <depend>autoware_msgs</depend>
```

Comment:
This doesn't exists.

Reply:
Fixed with: https://github.com/autowarefoundation/autoware_tools/pull/30/commits/5fc8fdb473d1054d5348485424fd61c14349747f